app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,8 @@
+#: app/templates/main/privacy.html:19
+msgid "Privacy notice"
```

Comment:
https://lga-2819-translation-access-cla.cloud-platform.service.justice.gov.uk/privacy - title didn't change for me, maybe I'm looking at an older version?

Reply:
Hiya Jav, i'm not having this issue, it's saying WELSH TEXT when i click the welsh.

Reply:
updated the title to so that it also translated